docs/overview.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+### Terminology
+- **Users** - end-users that want to transfer HBAR or HTS tokens from Hedera to EVM-based chain or Wrapped HBAR and Wrapped Tokens from EVM-based chain to Hedera
+- **Token Developers** - the developers of HTS tokens
+- **Validators** - parties/entities that are running the Validator node. They are providing authorisation for the minting and burning of wrapped tokens on the EVM-based chain as-well as transferring wrapped tokens back to Hedera.
+- **WHBAR** - ERC20 token issued and operated by Bridge validators. The token represents "wrapped" HBAR on the EVM-based chain. In other words we can say that Hbar is the `native` asset and `WHBAR` is the `non-native` asset.
+- **WHTS** - ERC20 token issued and operated by Bridge validators. The token represents "wrapped" HTS token on the EVM-based chain. In this case, `HTS` token is a `native` one and `WHTS` token is `non-native`.
+
+### Governance
+There are 2 setups for both of the Networks. On one side, there are 2 Hedera Bridge Accounts which are `n/m` threshold acccounts. Each validator has a Hedera compatible private key - 1 out of `m` that has `1/m` control over those threshold accounts.
+From now on, we will refer to them as `Bridge` and `Fee` accounts.
+
+The setup on the EVM chain is the same - [Gnosis MultiSig](https://github.com/gnosis/safe-contracts) is to be used with the same `n/m` threshold configuration. Each validator has an EVM compatible private key - 1 out of `m` that as `1/m` control over the threshold account.
+The Gnosis Multisig is configured as owner of:
+- Wrapped tokens deployed on the EVM chain (f.e WHBAR / WHTS tokens)
+- The Router smart contract
+
+#### Adding / Removing Members
+Validators can add new members or remove members from the validator set. We expect validators to have an off-chain communication channel using which validators can discuss the current setup, vote on removing validators or adding new ones.
```

Comment:
It is important to mention that adding a new member would require to have a validator instance, running with the provided credentials.